docs/scenarios-workflow.md

```diff
@@ -0,0 +1,65 @@
+# Scenario Tests Guide
+## Overview
+Our existing scenario tests are under `src\scenarios` in this repo, where each subdirectory contains a test asset that can be combined with a specific set of commands to do measurements. Currently we have scenario tests for [SDK](link), [Crossgen](link), [Blazor](link) and [other scenarios](). 
+<br>
+<br>
```

Comment:
Why `<br>`? I think the common markdown thing is to end the previous line with spaces and then just have a blank line.

Reply:
This approach does not work for me.. Both `&nbsp;` and `<br>` work though. Does it matter which interpreter is used?